resources/dicts/patrols/plains/hunting/leaf-bare.json

```diff
@@ -3783,86 +3783,74 @@
+        "exp": 25,
+        "weight": 20, 
+        "prey": ["medium"],
+        "stat_skill": ["HUNTER,1","SENSE,2","CLEVER,2"],
+        "relationships": [
+            {
+                "cats_to": ["s_c"],
+                "cats_from": ["patrol"],
+                "mutual": false,
+                "values": ["respect, trust, platonic"],
+                "amount": 5
+            } ] }, 
+        {
             "text": "The whole patrol stares at the sight of a fox, something cats fear, stuck upside with its whole rump flailing up and undignified. r_c can't stop {PRONOUN/r_c/self} from snorting at the sight. Then the whole patrol is laughing. For this one moment, your cats forget about the cold in the fur and hunger in their bellies to enjoy a moment with their friends.",
-                    "exp": 5,
-                    "weight": 20, 
-                    "prey": ["small"],
-                    "relationships": [
-                        {
-                            "cats_to": ["r_c"],
-                            "cats_from": ["patrol"],
-                            "mutual": true,
-                            "values": ["platonic"],
-                            "amount": 15
-                        }
-            ]
-            } ],
-            "fail_outcomes": [
-                {"text": "r_c snarls. {PRONOUN/r_c/subject/CAP} {VERB/r_c/is/are} too cold and too hungry to watch a stupid fox. This will be a hard hunt and if you can't focus, then go huddle with the kits at camp. ",
-                "exp": 0,
-                "weight": 15, 
-                "prey": ["very small"],
+                "exp": 5,
+                "weight": 20, 
+                "prey": ["small"],
                 "relationships": [
                     {
                         "cats_to": ["r_c"],
                         "cats_from": ["patrol"],
                         "mutual": true,
-                        "values": ["platonic, respect, trust"],
-                        "amount": -5
+                        "values": ["platonic"],
+                        "amount": 5
                     }
-                ]
-                    } 
-                
-    ]
-    }
+        ]  }
+    ], 
+        "fail_outcomes": [
+            {"text": "r_c snarls. {PRONOUN/r_c/subject/CAP} {VERB/r_c/is/are} too cold and too hungry to watch a stupid fox. This will be a hard hunt and if you can't focus, then go huddle with the kits at camp.",
+            "exp": 0,
+            "weight": 15, 
+            "prey": ["very small"],
+            "relationships": [
+                {
+                    "cats_to": ["r_c"],
+                    "cats_from": ["patrol"],
+                    "mutual": true,
+                    "values": ["platonic, respect, trust"],
+                    "amount": -5
+                }
             ]
         }
     ]
```

Comment:
what's this square bracket doing down here?

Reply:
@Tiritirimatangi  Vstudio is insisting that be there or the whole thing errors out